mcfix_analytic/__manifest__.py

```diff
@@ -1,12 +1,14 @@
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
+# Copyright 2018 Creu Blanca
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
 {
     'name': 'Multi Company Fix Analytic',
     'version': '11.0.1.0.0',
     'summary': 'Analytic fixes',
     'author': 'Eficent, Odoo Community Association (OCA)',
-    'website': 'http://www.eficent.com',
+    'website': 'https://github.com/OCA/multi-company',
     'license': 'LGPL-3',
-    'depends': ['analytic', 'mcfix_mail'],
+    'depends': ['analytic', 'mcfix_base'],
```

Comment:
@etobella @mreficent not sure why there was a dependency with mcfix_mail. Seems that it only makes sense to keep with mcfix_base. Any thoughts?